force-app/main/default/lwc/platformDevCertCalculator/platformDevCertCalculator.html

```diff
@@ -0,0 +1,83 @@
+<template>
+    <lightning-card title="PD1 Calculator">
+        <!-- Process-->
+        <lightning-button 
+            variant="brand" 
+            label="Calculate" 
+            title="Calculate" 
+            slot="actions" 
+            onclick={calculateScore}
+        ></lightning-button>
+        <!-- Input Information -->
+        <div class="slds-var-p-horizontal_small">
+            <lightning-input
+                type="number"
+                name="devFundamentals"
+                label="Developer Fundamentals: 23%"
+                value={devFundamentalScore}
+                onchange={handleChange}
```

Comment:
Try adding min and max attributes so that they users can't enter incorrect values.